meta/autoid/errors.go

```diff
@@ -30,9 +30,7 @@ var (
 
 const (
 	// AutoRandomPKisNotHandleErrMsg indicates the auto_random column attribute is defined on a non-primary key column, or the table's primary key is not a single integer column.
-	AutoRandomPKisNotHandleErrMsg = "column %s is not the single integer primary key, or alter-primary-key is enabled"
-	// AutoRandomExperimentalDisabledErrMsg is reported when the experimental option allow-auto-random is not enabled.
-	AutoRandomExperimentalDisabledErrMsg = "auto_random is an experimental feature, which can only be used when allow-auto-random is enabled. This can be changed in the configuration."
+	AutoRandomPKisNotHandleErrMsg = "column %s is not the integer primary key, or table is created with alter-primary-key enabled"
```

Comment:
is there any place checked with this `table is created with alter-primary-key enabled`?

Reply:
https://github.com/pingcap/tidb/blob/2bb1ff5a1efaec0cb38b575309091254df19d0b3/ddl/serial_test.go#L843

Reply:
got, 
there is not explicit AlterPrimaryKey judgement, when this is on, pk is not handle indeed.